en/docs/develop/product-apis/publisher-apis/publisher-v1/publisher-v1.yaml

```diff
@@ -7895,17 +8262,17 @@ definitions:
         type: array
         items:
           type: string
-          example: ["Pizzashack, Petstore"]
+        example: []
```

Comment:
examples?

Reply:
Gateway labels have to be predefined before attaching to an API, in case we define a non-existent Gateway label in the API payload, that label gets skipped. Hence left the field as empty to avoid confusion.